### PR TITLE
feat(MonitoringTool): Inspect of arbitrary model found by objectName

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

This utility may be helpful for debugging complex data flows in the application.

Once a model has `objectName` assigned, it can be inspected in runtime via `Monitoring Tool`. Object name and root object should be provided.

For models not created in singletons good root object is `applicationWindow`.
For models created in singletons, singleton handler must be provided, as explained in the tool's UI.

Closes: #14971

### Affected areas
`MonitoringTool`

### Screenshot of functionality (including design for comparison)

[Screencast from 29.05.2024 12:01:50.webm](https://github.com/status-im/status-desktop/assets/20650004/0e1e3a16-5146-4db9-ae9e-8f664fce1e12)

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

File: monitoring/ModelInspectionPane.qml [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+        MenuSeparator {
+            Layout.fillWidth: true
+        }
+
+        Label {
+            visible: showControls
+
+            text: "Hint: use right/left button click on a column " +
+                  "header to ajust width, press cell content to " +
```
Comment (caybro) [inline monitoring/ModelInspectionPane.qml:89]:
```suggestion
                  "header to adjust width, press cell content to " +
```

File: monitoring/MonitorEntryPoint.qml [hunk 2]
```diff
@@ -1,502 +1,375 @@
...
+
+        TabBar {
+            id: tabBar
+            Layout.fillWidth: true
+            Layout.bottomMargin: 10
 
-                text: "Context properties:"
-                font.bold: true
+            TabButton {
+                text: qsTr("Context properties inspection")
```
Comment (caybro) [inline monitoring/MonitorEntryPoint.qml:29]: No need for `qsTr` here :)

File: monitoring/MonitorEntryPoint.qml [hunk 3]
```diff
@@ -1,502 +1,375 @@
...
-                    }
-
-                    MenuSeparator {
                         Layout.fillWidth: true
+                        wrapMode: Text.Wrap
+
+                        text: "Note: 'applicationWindow' is good root object in"
+                              + " most cases. 'WalletStores.RootStore' and"
+                              + " `SharedStores.RootStore` are also exposed for"
+                              + " covenience for models created within those singletons."
```
Comment (caybro) [inline monitoring/MonitorEntryPoint.qml:301]:
```suggestion
                              + " convenience for models created within those singletons."
```

**Review by caybro — APPROVED** [review]
Great stuff!

Just some minor typos

**Pull request merged**